metric_config_parser/metric.py

```diff
@@ -324,7 +344,7 @@ def from_dict(cls, d: dict) -> "MetricsSpec":
                 {"name": k, **dict((kk.lower(), vv) for kk, vv in v.items())}, MetricDefinition
             )
             for k, v in d.items()
-            if k not in known_keys and k != "28_day"
+            if k not in known_keys and k not in ("28_day", "28_day_preenrollment")
```

Comment:
I'm guessing that `"28_day"` is called out specifically because that is what `days28` maps to in the `mozanalysis_label` section above, so shouldn't be necessary here. If that's not the case, then I'm not sure where `"28_day"` (or `"28_day_preenrollment"`) would come from.

If my guess is correct, I suppose it doesn't hurt to have this here still, just could be confusing in the future.

Reply:
Yeah, I wasn't sure if the new category should conform to the existing pattern, which might just be there for legacy support. By adding this new category, we could break that pattern if desired. What do you think @mikewilli ?

Reply:
Agree, removed